### PR TITLE
Add f3 to halloffame and README

### PR DESCRIPTION
**Pull request by error401de**
As discussed in https://github.com/geokrety/geokrety-website/pull/514 another PR for the new theme
Can you please review @kumy @boly38 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,4 +10,8 @@ Please contact us directly for any security issue contact@geokretymap.org
 
 cf. [INSTALL.md](INSTALL.md) and [CONTRIBUTING.md](CONTRIBUTING.md) readme files.
 
+# Credits
+
+![powered by F3](https://cdn.geokrety.org/images/logos/f3.png)\
```
Comment (kumy) [inline README.md]: `svg`, no ?

Comment (error401de) [reply README.md]: oh man, sorry, copied from the wrong commit... will change

Comment (kumy) [reply README.md]: Don't forget to squash your commits into only one please.

**Comment by error401de** [comment]
@kumy Okay, it's now only one commit :)
Thanks for your patience, I learned a lot about git with this PRs!

**Comment by kumy** [comment]
@error401de thanks for your time! Enjoy your hacktoberfest!

**Comment by boly38** [comment]
thanks @error401de  👍

**Comment by kumy** [comment]
@error401de you may see the result at https://github.com/geokrety/geokrety-website/tree/feature/new-theme and https://new-theme.staging.geokrety.org/en/hall-of-fame.

The logo size and disposition may need to be resized a bit ;)

**Comment by kumy** [comment]
> The logo size and disposition may need to be resized a bit ;)

… on the README page.

**Comment by error401de** [comment]
> > The logo size and disposition may need to be resized a bit ;)
> 
> … on the README page.

True, see https://github.com/geokrety/geokrety-website/pull/518 :)

**Pull request merged**